content/en/docs/reference/glossary/cluster-registry.md

```diff
@@ -0,0 +1,18 @@
+title: Cluster Registry
+<<<<<<< HEAD:content/en/docs/reference/glossary/cluster-registry.md
```

Comment:
This looks like an unfinished merge. I recommend fixing that and squashing / rebasing.